ibis/backends/polars/tests/conftest.py

```diff
@@ -0,0 +1,73 @@
+from pathlib import Path
+
+import pytest
+
+import ibis
+import ibis.expr.types as ir
+from ibis.backends.tests.base import BackendTest, RoundAwayFromZero
+from ibis.backends.tests.data import array_types
+
+pl = pytest.importorskip("polars")
+
+
+class TestConf(BackendTest, RoundAwayFromZero):
+    bool_is_int = True
```

Comment:
Is this correct? Does polars not support a proper bool type?

Reply:
It dies, fixing.

Reply:
Updated.